src/Sulu/Bundle/AdminBundle/Resources/js/components/ColumnList/tests/Toolbar.test.js

```diff
@@ -69,10 +68,9 @@ test('Should close dropdown when item is clicked', () => {
 
     const toolbar = mount(<Toolbar toolbarItems={toolbarItems} />);
 
-    expect(toolbar.find(ToolbarDropdownListOption)).toHaveLength(0);
+    expect(toolbar.find('ToolbarDropdown').find('Action')).toHaveLength(0);
     toolbar.find(ToolbarDropdown).find('a').simulate('click');
-    expect(toolbar.find(ToolbarDropdownListOption)).toHaveLength(2);
-
```

Comment:
Would also leave this empty line, to indicate that there are two different cases in this test.